src/redux/actions/differentialExpression/loadDifferentialExpression.js

```diff
@@ -68,7 +69,7 @@ const loadDifferentialExpression = (
       payload: {
         experimentId,
         data: rows,
-        cellSets,
+        cellSets: comparisonGroup,
```

Comment:
Why is `comparisonGroup` passed as `cellSets`? Is it a big problem to unify this naming?
Also about `comparisonGroup`, I think I would call it `comparisonCellSets` or something like that perhaps.

Reply:
It should be `comparisonGroup`, because stores the, not really the `cellSets`. It's confusing to have it named as `cellSets` when we have another object named `cellSet`. I'll rename it and correct other users of `cellSets` for differential expression.

Reply:
no, it should be `comparisonGroup`. It's confusing to use the term `cellSet` because we already use it for another object in the platform. I'll rename the keys to `comparisonGroup`.

Reply:
moved to use `comparisonGroup`